package.json

```diff
@@ -174,6 +174,7 @@
     "geco": "0.11.1",
     "hammerjs": "2.0.8",
     "html-webpack-harddisk-plugin": "1.0.1",
+    "ios-sim": "8.0.1",
```

Comment:
why you import this dep?

Reply:
yarn ios:run a planté si je faisais pas ca.